README.md

```diff
@@ -40,26 +40,31 @@ language.
 
 Install dependencies:
 
-* meson
-* wayland
-* wayland-protocols
-* EGL
-* GLESv2
-* libdrm
-* GBM
-* libinput
-* xkbcommon
-* udev
-* pixman
+* [meson]
+* [Wayland], namely:
+  * [libwayland]
+  * [wayland-protocols]
+* [EGL], namely:
+  * [libglvnd]
```

Comment:
libglvnd is not required. Plain Mesa works fine too (and arguably works better, because one less component in the chain which might not support and ext or feature). We need an EGL implementation, and there are many.